packages/StackedBar/src/StackedBar.stories.tsx

```diff
@@ -207,6 +207,47 @@ export const CustomRangeAndFormat = {
   },
 };
 
+export const ZeroFirstAndLast = {
```

Comment:
j'ai pas QA encore, mais j'ai comme le feeling que la solution que tu as supporterais pas que les 2-3 premiers soit à 0, un chiffre ds le millieu + 2-3 dernier à 0?

Reply:
it doesn't work if all are 0 except for the last:
![image](https://github.com/gsoft-inc/ov-igloo-ui/assets/110498164/99d79670-5dca-418e-a1eb-7967f24b349c)

Reply:
Same when all the ends are 0 except for the first: 
![image](https://github.com/gsoft-inc/ov-igloo-ui/assets/110498164/210d2622-cdb2-41f6-86b3-66045abd21e4)

Reply:
Good catch, going back to the drawing board.

Reply:
not sure why it wasn't like that to begin with... maybe because the tooltip used to be instead the component and you couldn't have overflow hidden. but if it's easy to do, do it that way.

Reply:
Yes that was not possible, I'm going to do this in JS.